.template/addons/github/.github/workflows/deploy.yml.tt

```diff
@@ -0,0 +1,69 @@
+name: Deploy
+
+on:
+  workflow_run:
+    workflows:
+      - Test
+    branches:
+      - master
```

Comment:
it should be the `main` branch to adapt to the change from Github

Reply:
yes, I added `main` but I think we also keep `master` as we can apply this add-on to the existing project